Makefile

```diff
@@ -2,10 +2,19 @@
 all: buildroot/Makefile buildroot/.config
 	make -C buildroot
 
+clean:
+	BR2_EXTERNAL=.. make -C buildroot clean
+
 buildroot/Makefile:
```

Comment:
Good idea, but this should go into a separate pull request. After all, this PR has the title "Add release to Makefile", and a "clean" target is out of scope here...